src/misc.ts

```diff
@@ -29,9 +37,14 @@ export const items: Map<string, [vscode.CompletionItemKind, vscode.MarkdownStrin
             vscode.CompletionItemKind.Keyword,
             new vscode.MarkdownString(
                 `
-                \`if_assigned\` docs
+                syntax: 
+                if_updated <{:msg "property description"}> propertyCondition;
```

Comment:
```suggestion
                if_assigned <{:msg "property description"}> propertyCondition;
```
I guess the `if_assigned` is supposed here.